src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -47,7 +47,7 @@ def load_arguments(self, _):
             c.argument('timeout', options_list=['--timeout'], type=int, default=3600,
                        help='Time in seconds for the copy operation to finish. Increase this time if '
                        'you are going to copy large images (disks) like 512GB or more.')
-            c.argument('temporary_resource_group_name', options_list=['--temporary_resource_group_name'],
+            c.argument('temporary_resource_group_name', options_list=['--temporary-resource-group-name'],
```

Comment:
break change ?

Reply:
The old name is wrong. It is a breaking change, but it should be updated.